src/main/java/net/minecraftforge/server/command/PortallessTeleporter.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Minecraft Forge
```

Comment:
Remove, replace with lambda, `(_,_,_,_,repos) -> repos.apply(false)`

Reply:
This is not possible since ITeleporter would need to be a functional interface then, and functional interfaces are not allowed to have default methods.

Reply:
Should I increase patch size of the Teleporter class to make it a functional interface?

Reply:
Lambda should be possible if the interface is a single method even with a default. I'd have to check later but I could of sworn that I've done that before... But the point is I dont want this 'PortalessTeleporter' class as public api we have to maintain.

Reply:
unterstandable, would a anonymous inner class be better?
so something like
```
private static ITeleporter portalless = new ITeleporter(){
    @Override
    public Entity placeEntity(Entity entity, ServerWorld currentWorld, ServerWorld destWorld, float yaw, Function<Boolean, Entity> repositionEntity) 
    {
        return repositionEntity.apply(false);
    }
}
```

Reply:
Now it is no longe rpublic api.